### DISCUSSION TITLE
Commit status update via api v4

### DISCUSSION THREAD

**Discussion by Jkovarik** [community/community#24543] [Apps, API and Webhooks]
<p>I’m looking at adding an API update to commit statuses from a CI server via the status API - is this something that’s supported (or planning on being supported) or should I utilize v3 for now?   </p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/jkovarik">@jkovarik</a>,</p>
<p>Thanks for reaching out! The <a href="https://developer.github.com/v4/object/status/" rel="nofollow noopener">status Object</a> is immutable, so to update this you will need to create a new status. At this point in time, there is no GraphQL mutation to do so, and you will need to use the <a href="https://developer.github.com/v3/repos/statuses/#create-a-status" rel="nofollow noopener">Create a Status REST API Endpoint</a> instead.</p>
<p>Cheers,</p>

**Comment by warsaw** [comment]
<p>I wanted the same thing.  For now I’m creating statuses using the v3 API.</p>

**Comment by kquick** [comment]
<p>I would like this ability via v4 as well (not sure where else to upvote this).</p>
